javascript/organizing_your_javascript_code/factory_functions_and_module_pattern.md

```diff
@@ -242,7 +242,9 @@ const Nerd = (name) => {
 }
 ~~~
 
-- Before moving on have a look at [this](https://medium.com/javascript-scene/3-different-kinds-of-prototypal-inheritance-es6-edition-32d777fa16c9) article. In the second half of the article, the author goes into some things that we aren't really talking too much about here, but you'll be rewarded if you spend some time figuring out what he's talking about. Good stuff!
+- Before moving on have a look at this article. That's a tough one but you will get rewarded if you spend some time with it, Good Stuff!
+A notice about backbone.js, don't spend much time on it, [this video](https://www.youtube.com/watch?v=RBgm7Zgv2dM) should suffice.
```

Comment:
Can we move this to additional resources?  It is by no means required for students to know anything about Backbone.js... and not worth wasting their 20 minutes.  I'd put a statement like you did for the Object.assign statement above.

"This article mentions backbone.js, a relatively old javascript framework.  You don't need to know anything about it to understand the article, but if you're curious, you can find a video on it in the additional resources section"

Reply:
might I put in an extra request for the link to be more descriptive 😄?

Reply:
yes! concur with  @ManonLef